systems-management/nisysmgmt.yml

```diff
@@ -1119,10 +1149,9 @@ definitions:
         description: The name of the package.
         example: "ni-securityupdate-kb67l8lcqw-killbits"
       plugin:
-        type: integer
-        format: int32
+        type: string
         description: Agent that will be used to install the package.
-        example: 2
+        example: "wininst"
```

Comment:
Should this be an enum, or do we expect clients to be able to send custom values?

Reply:
Both this and the next are some changes Eliza did, and she said those are open-ended strings. Some of them are integers, some of them strings. She fixed this when she saw some exceptions for certain packages that contains priority = 'wininst' or 'standard'. It's best to let it a string, since we are not really interpreting those, but just read them from the system and feed them to nipkg or opkg.